react-app/src/native/components/leagues/TeamList.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Text, Card, CardItem} from 'native-base';
+import { Table, Row } from 'react-native-table-component';
+import { StyleSheet } from 'react-native';
+import Loading from '../common/Loading';
+
+export default class Schedule extends React.Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+
+        const { league, leagueName, lookups } = this.props;
+
+        const flexArr = [2, 7];
+        const teamTable = {
+            header: ['Team #', 'Name'],
+            data: [],
+        }
+
+        league.teams.map((team, i) => {
+            teamTable.data.push([team.numInLeague, team.name]);
+        });
+
+        if (league == null || league.isFetching) return <Loading />
+
+        return (
+            <Card>
+                <CardItem header>
+                    <Text>{leagueName}</Text>
+                </CardItem>
+                <CardItem cardBody style={{ padding: 10 }}>
+                    <Table style={{ flex: 1, marginBottom: 10 }} borderStyle={{ borderWidth: 0, borderColor: "transparent" }}>
+                        <Row
+                            flexArr={flexArr}
+                            data={teamTable.header}
+                            style={styles.header}
+                            textStyle={styles.headerText}
+                        />
+                        {
+                            teamTable.data.map((rowData, index) => (
+                                <Row
+                                    key={index}
+                                    flexArr={flexArr}
+                                    data={rowData}
+                                    style={[styles.row, index % 2 == 1 && { backgroundColor: '#e6e6e6' }]}
+                                    textStyle={styles.text}
+                                />
+                            ))
+                        }
+                    </Table>
+                </CardItem>
+            </Card>
+        );
+    }
+}
+
+const styles = StyleSheet.create({
+    container: { flex: 1, backgroundColor: 'yellow' },
+    header: { padding: 2, borderBottomWidth: 2, borderBottomColor: 'black' },
+    headerText: { fontWeight: "bold" },
+    text: {},
+    row: { padding: 2 }
+});
+
+/**
+ * 
+ *  {
+                    league != null && !league.isFetching &&
+                    <Content><Text>{JSON.stringify(league.leagueSchedule, null, 2)}</Text></Content>
+                }
```

Comment:
Remove this comment. If you need to see whats in the league object use console.log at the top of the render function and check the console.